root/iswc/Index.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+
+const {withCatalystContext} = require('../context');
+const Layout = require('../layout');
+const ArtistCreditLink = require('../static/scripts/common/components/ArtistCreditLink');
+const CodeLink = require('../static/scripts/common/components/CodeLink');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const {l, ln} = require('../static/scripts/common/i18n');
+import {lp_attributes} from '../static/scripts/common/i18n/attributes';
+import {l_languages} from '../static/scripts/common/i18n/languages';
+import loopParity from '../utility/loopParity';
+
+type PropsT = {|
+  +$c: CatalystContextT,
+  +iswcs: $ReadOnlyArray<IswcT>,
+  +works: $ReadOnlyArray<WorkT>,
+|};
+
+const Index = ({$c, iswcs, works}: PropsT) => {
+  const userExists = $c.user_exists;
+  const iswc = iswcs[0];
+  return (
+    <Layout fullWidth title={l('ISWC “{iswc}”', {iswc: iswc.iswc})}>
+      <h1>
+        {l('ISWC “{iswc}”',
+          {__react: true, iswc: <CodeLink code={iswc} key="iswc" />})}
+      </h1>
+      <h2>
+        {ln(
+          'Associated with {num} work',
+          'Associated with {num} works',
+          works.length,
+          {num: works.length},
+        )}
+      </h2>
+      <form action="/work/merge_queue" method="post">
+        <table className="tbl">
+          <thead>
+            <tr>
+              {userExists ? (
+                <th style={{width: '1em'}}>
+                  <input type="checkbox" />
+                </th>
+              ) : null}
+              <th>{l('Title')}</th>
+              <th>{l('Writers')}</th>
+              <th>{l('Artists')}</th>
+              <th>{l('Type')}</th>
+              <th>{l('Language')}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {works.map((work, index) => (
+              <tr
```

Comment:
This duplicates 95% of `buildResult` in root/search/components/WorkResults.js so it would be nice to create a reusable component for both pages to share.

You'll just need to add props to hide the iswcs column and possibly the merge checkboxes.

Reply:
To clarify, the [WorkResults.js in your search PR](https://github.com/metabrainz/musicbrainz-server/pull/716/files#diff-b63869045ac7691d7b0edeedbfed8784R29) (forgot it's not visible here).